### PR TITLE
[yugabytedb] update tc url form and jdbc doc page

### PR DESCRIPTION
**Pull request by srinivasa-vasu**
update tc URL and JDBC doc page for yugabytedb

### CODE REVIEW

File: docs/modules/databases/jdbc.md [hunk 1]
```diff
@@ -55,6 +55,11 @@ Insert `tc:` after `jdbc:` as follows. Note that the hostname, port and database
 
 `jdbc:tc:tidb:v6.1.0:///databasename`
 
+#### Using YugabyteDB
+
+`jdbc:tc:yugabytedb:2.14.4.0-b26:///databasename`
```
Comment (eddumelendez) [inline docs/modules/databases/jdbc.md:60]: We can keep `yugabyte`. We can revisit on the next major version.

**Review by eddumelendez — CHANGES_REQUESTED** [review]
Thanks for the PR! Let's just update the docs and consider the change in the JDBC url for the next major version.

File: docs/modules/databases/jdbc.md (hunk 1 above)
Comment (eddumelendez) [reply docs/modules/databases/jdbc.md:60]: @srinivasa-vasu can you update the PR taking into account the suggestions, please?

Comment (srinivasa-vasu) [reply docs/modules/databases/jdbc.md]: @eddumelendez - Apologies, my bad, totally missed it. Reverted just to the doc update. Can we update the `tc url` to `yugabytedb` in the next major release?

**Review by eddumelendez — APPROVED** [review]

**Comment by eddumelendez** [comment]
Thanks @srinivasa-vasu ! This is now in `main` branch.

**Pull request merged**